### ISSUE TITLE
Low contrast ratio

### ISSUE THREAD

**Issue by fflaten** [pester/docs#221]
Certain colors should be changed to have a better contrast ratio for readability and accessibility.
Some of these are shown by Lighthose in Chromium.

- **Dark mode:** Active link
  ![image](https://user-images.githubusercontent.com/3436158/182478221-4c6b5d3f-48d4-45c9-9816-7f5ce3849e78.png)
- **Both:** Active section link
  ![image](https://user-images.githubusercontent.com/3436158/182478388-d2c36805-182e-4216-b429-74404550eb39.png)

- **Light mode:** The pink and especially the turquoise color against the grey background used in the prism-theme (code blocks)
- **Light mode:** The active page in the breadcrumbs and menu. Changing the background (`--ifm-hover-overlay` css-variable) slighty from 5% -> 4% alpha is an option.
- **Light mode:** Placeholder-text in DocSearch. Could possibly ignore this.

**Comment by fflaten** [comment]
Related PR fixes everything above except prism-theme.
